unit/jest-test/registrationForm.spec.ts

```diff
@@ -0,0 +1,137 @@
+
+import { RegistrationForm } from '../src/registration-form';
+import {
+  NameError,
+  PasswordError,
+  RepeatPasswordError,
+  validName,
+  validEmail,
+  invalidName,
+  invalidEmail,
+  invalidRepeatPassword,
+  invalidShortPassword,
+  validPasswordMoreThenEightCharacters,
+  validNameWithSpaces,
+  validPasswordEightCharacters,
+  invalidPasswordWithoutSpecialSymbol,
+  invalidPasswordWithoutCapital,
+  invalidPasswordWithoutdigit,
+  EmailError
+} from '../src/constants/constants';
+
+describe('RegistrationForm Tests', () => {
+  let registrationForm: RegistrationForm;
+
+  beforeEach(() => {
+    registrationForm = new RegistrationForm(
+      validName,
+      validEmail,
+      validPasswordMoreThenEightCharacters,
+      validPasswordMoreThenEightCharacters
+    );
+  });
```

Comment:
This precondition only fits for the first test. Please add this to the first test and delete `beforeEach()` block.

Reply:
done